rest/src/routes/accountRoutes.js

```diff
@@ -80,5 +84,35 @@ module.exports = {
 				next();
 			});
 		});
+
+		// CMC specific endpoint
+		const readAndParseNetworkPropertiesFile = () => {
+			const readFile = util.promisify(fs.readFile);
+			return readFile(services.config.apiNode.networkPropertyFilePath, 'utf8')
+				.then(fileData => ini.parse(fileData));
+		};
+
+		// CMC specific endpoint
+		server.get('/network/currency/supply/circulating', (req, res, next) => readAndParseNetworkPropertiesFile()
```

Comment:
This endpoint should only be visible in mainnet? For private and testnet, seems to just returned nothing.

Reply:
Hmm, tested it on private net, works, but the unclaimed total is 0 which is expected, will try again